<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputFileProcess</class>
 <widget class="QWidget" name="OutputFileProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>General settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSize">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>9</number>
        </property>
        <item row="2" column="0">
         <widget class="QPushButton" name="buttonProcessNow">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Process now</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buttonSendToBatch">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Send to batch</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioProcessSequence">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Process selected sequence</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioProcessAll">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Process the entire video</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLength">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Length:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelFile">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="buttonBrowse">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineFileName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="buttonSetOptimal">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Set optimal size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSpinBox" name="spinWidth">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1600000000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelX">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>x</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinHeight">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1600000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="TimeSequenceWidget" name="sequenceTime" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeSequenceWidget</class>
   <extends>QWidget</extends>
   <header>../src/widgets/timesequence.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineFileName</tabstop>
  <tabstop>buttonBrowse</tabstop>
  <tabstop>spinWidth</tabstop>
  <tabstop>spinHeight</tabstop>
  <tabstop>buttonSetOptimal</tabstop>
  <tabstop>radioProcessSequence</tabstop>
  <tabstop>buttonProcessNow</tabstop>
  <tabstop>buttonSendToBatch</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
